classes_objects.py

```diff
@@ -0,0 +1,23 @@
+# define the Vehicle class
+class Vehicle:
+    name = ""
+    kind = "car"
+    color = ""
+    value = 100.00
+
+    def __init__(self, name: str, color: str, kind: str, value: float) -> None:
+        self.name = name
+        self.color = color
+        self.value = value
+        self.kind = kind
+        
+    def description(self) -> None:
```

Comment:
```suggestion
    @property
    def description(self) -> str:
```
The return. type. for this method would be a `str`.

Also, I know it hasn't been covered in any of these tutorials yet (it's part of the advanced tutorials you'll do later) but just to share some knowledge, [In general, computed attributes should be made using the @property decorator.](https://realpython.com/python-property/#providing-computed-attributes)

for now though I suggest correcting the type annotation

Reply:
done, only fixed the type annotation.